packages/theme/src/theme/_colors.scss

```diff
@@ -3,93 +3,129 @@
 /// @group Colors
 ////
 
-/// Primary color, mostly used for the selected elements
-/// and to draw attention to the important elements of interaction.
-///
+////
+/// Primary colors
+/// These colors defines the brand color
+/// @group PrimaryColors
+////
+
+/// Regal Blue
 /// @type Color
-$st-tropaz: #266092;
+$regal-blue: #17486E;
 
-/// Secondary color
-///
+/// Saint Tropaz
 /// @type Color
-$rio-grande: #C3D600;
+$st-tropaz: #236192;
 
-/// Contextual color for informational alert messages
+/// Green Talend
 ///
 /// @type Color
-$scooter: #66BDFF;
+$green-talend: #B6BE00;
 
-/// Linked to valid/validated/validation elements.
-///
+////
+/// Accent colors
+/// These colors emphasizes actions and highlighted information
+/// @group AccentColors
+////
+
+/// Black
 /// @type Color
-$limeade: #8ABC00;
+$black: #202020;
 
-/// Used for validation message in inputs
-///
+/// Dove Gray
 /// @type Color
-$pirate-gold: #B78001;
+$dove-gray: #555;
 
-/// Used to notify invalid elements or warning information.
-///
+/// Dark Silver
 /// @type Color
-$lightning-yellow: #F2DA0D;
+$dark-silver: #888;
 
-/// Used for errors.
-///
+/// Gray
 /// @type Color
-$chestnut-rose: #E96065;
+$gray: #969696;
 
-/// Used for labels, titles.
-///
+/// Silver Chalice
 /// @type Color
-$black: #202020;
+$silver-chalice: #ABABAB;
 
-/// Used for some labels
-///
+/// Silver
 /// @type Color
-$mine-shaft: #333;
+$silver: #C6C6C6;
 
-/// Used for subtitles.
-///
+/// Rio Grande
 /// @type Color
-$dove-gray: #555;
+$rio-grande: #C3D600;
 
-/// Used for some labels
-///
-/// @type color
-$dark-silver: #888;
+/// Scooter
+/// @type Color
+$scooter: #66BDFF;
 
-/// Used for backgrounds.
-///
+/// Chesnut Rose
 /// @type Color
-$silver: #C6C6C6;
+$chestnut-rose: #E96065;
 
-/// Used for input disabled colors, etc..
-///
+/// Lightning Yellow
 /// @type Color
-$silver-chalice: #ABABAB;
+$lightning-yellow: #F2DA0D;
 
-/// Used for backgrounds etc.
-///
+////
+/// Background colors
+/// These colors used for backgrounds
+/// @group BackgroundColors
+////
+
+/// White
 /// @type Color
-$alto: #DEDEDE;
+$white: #FFF;
 
-/// Used for panels.
-///
+/// Concrete
 /// @type Color
 $concrete: #F2F2F2;
 
-/// Used for empty spaces, toolbar-panels etc.
-///
+/// Alto
 /// @type Color
-$white: #FFF;
+$alto: #DEDEDE;
+
+/// Gallery
+/// @type Color
+$gallery: #EAEAEA;
+
+/// Slate Gray
+/// @type Color
+$slate-gray: #758495;
+
+/// Wild Sand
+/// @type Color
+$wild-sand: #F6F6F6;
+
+////
+/// Help colors
+/// These colors used for helping purposes
+/// @group HelpColors
+////
 
 /// Used for default color theme
-///
 /// @type Color
-$smalt-blue:    #4F8992;
+$patina: #649083;
 
 /// Used for shadows
-///
 /// @type Color
 $shadow: rgba(0, 0, 0, 0.2);
+
+////
+/// Deprecated colors
+/// These colors are deprecated
+/// @group DeprecatedColors
+////
+
+/// Should be changed to $rio-grande
+/// @type Color
+$limeade: #8ABC00;
```

Comment:
Shouldn't we remove the constant $limeade (the previous SUCCESS_GREEN) to make sure it is not used anymore?